t/cli/test_admin.sh

```diff
@@ -112,6 +113,66 @@ if [ $count -eq 0 ]; then
     exit 1
 fi
 
+echo "
+apisix:
+    enable_admin: true
+    allow_admin:
+        - 127.0.0.0/8
+" > conf/config.yaml
+
+make init
+
+count=`grep -c "allow 127.0.0.0/8" conf/nginx.conf`
+if [ $count -eq 0 ]; then
```

Comment:
Not sure I understand what the purpose of using count here is? Worried about other configurations also 
`listen 127.0.0.0/8` ?

Reply:
`count` used to find the target address render or not. It'll place with `allow ip-address or cidr;`.

https://github.com/apache/apisix/blob/6ebe02797fb564c84b30df1865adef7f473880bf/apisix/cli/ngx_tpl.lua#L515-L528